etc/config/csharp.amazon.properties

```diff
@@ -7,16 +7,28 @@ defaultCompiler=dotnet700csharp
 group.csharp.compilers=dotnettrunkcsharp:dotnet700csharp:dotnet6011csharp
 
 compiler.dotnet6011csharp.exe=/opt/compiler-explorer/dotnet-v6.0.11/.dotnet/dotnet
-compiler.dotnet6011csharp.name=.NET 6.0.403
+compiler.dotnet6011csharp.name=.NET 6.0.110
 compiler.dotnet6011csharp.clrDir=/opt/compiler-explorer/dotnet-v6.0.11
 compiler.dotnet6011csharp.buildConfig=Release
 compiler.dotnet6011csharp.langVersion=latest
 
-compiler.dotnet700csharp.exe=/opt/compiler-explorer/dotnet-v7.0.0/.dotnet/dotnet
```

Comment:
Compiler ids should always be present if they have been deployes live, else old links will break

Reply:
this was "wrong" in first place. starting from this point, we will freeze them.

Reply:
added as alias